README.md

```diff
@@ -66,18 +66,54 @@ To achieve this MLOS provides:
 
     Once hooks are created in the target system, iteration on the external agent can be more rapidly developed and deployed.
 
-## Build
+
+## Python only installation
+
+Some of the examples require only the installation of the mlos Python library. These examples do not use the shared memory infrastructure.
+To use this simplified installation, it's recommended to use the Anaconda python distribution and create a new conda environment:
+
+```
+$ conda create -n mlos_environment
+$ conda activate mlos_environment
+```
+
+You can download the mlos code using git:
+
+```
+$ git clone git@github.com:microsoft/MLOS.git
```

Comment:
Should we include an https:// example as well?

Reply:
Should we do only one or the other?

Reply:
I think it depends on whether we want to encourage commits or not, right?

If it's only for consumption, then `https` is more cross platform and readily available without additional setup cost.

Should we just assume that if you're going to submit PRs, then you probably know how to fill in the details of either using a different clone URL or else updating the `git remote` afterwards?

Reply:
yeah probably, so let's do https.